en/config/actuators.md

```diff
@@ -0,0 +1,116 @@
+# Actuator Configuration and Testing
+
+After the airframe is setup you should configure the geometry, assign actuators to outputs, and test the actuator response.
+This can be done in *QGroundControl*, under the **Vehicle Setup > Actuators** tab.
+
+:::note
+This view is only enabled if dynamic control allocation is enabled via [SYS_CTRL_ALLOC](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC), which will become the default in future.
+If you enable it, also make sure to select the airframe with [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME).
+It replaces the existing mixer files and allows configuration via parameters instead.
+:::
+
+This is a screenshot for a multicotor:
+
+![Actuators MC (QGC)](../../assets/config/qgc_actuators_mc_aux.png)
+
+
+:::note
+If you find a setting missing (e.g. reversable motors), make sure to enable the **advanced** checkbox in the top right corner (not visible above).
+:::
+
+## Geometry
+The airframe pre-selects the geometry type, which can provide additional configuration options, such as the motors positions on a multicotor, or the control surface types on a plane.
+The UI reflects that and displays a customized view for each type.
```

Comment:
Ah, so you're saying that the geometry options you are offered will depend on the airframe - so you'll just be offered the multicopter option here?

What I'm getting is where the available geometry is configurable - ie at this point it is too late to select a + config right - the airframe selection means here you already have to have chosen X.

Reply:
Yes. We have some flexibility where that line exactly is. For now it is: when selecting an MC you can fully reconfigure the motors: switch between X/+/h, add/remove motors, change motor positions. What you cannot do is change it to a vtol for example.